client/app/pages/data-sources/show.js

```diff
@@ -3,12 +3,12 @@ import debug from 'debug';
 import template from './show.html';
 
 const logger = debug('redash:http');
-const deleteConfirm = { class: 'btn-warning', title: 'Delete' };
-function logAndToastrError(deleteObject, httpResponse, toastr) {
+export const deleteConfirm = { class: 'btn-warning', title: 'Delete' };
+export function logAndToastrError(deleteObject, httpResponse, toastr) {
   logger('Failed to delete ' + deleteObject + ': ', httpResponse.status, httpResponse.statusText, httpResponse.data);
   toastr.error('Failed to delete ' + deleteObject + '.');
 }
-function toastrSuccessAndPath(deleteObject, deletePath, toastr, $location) {
+export function toastrSuccessAndPath(deleteObject, deletePath, toastr, $location) {
```

Comment:
What's the reason for this?

Reply:
Fix import/named
```
/path/to/redash/client/app/pages/destinations/show.js
  3:10  error  deleteConfirm not found in '../data-sources/show'         import/named
  3:25  error  logAndToastrError not found in '../data-sources/show'     import/named
  3:44  error  toastrSuccessAndPath not found in '../data-sources/show'  import/named
```

https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/named.md